dashbuilder-client/dashbuilder-dataset-editor/src/main/java/org/dashbuilder/dataset/editor/client/screens/DataSetDefWizardScreen.java

```diff
@@ -216,10 +216,10 @@ void onSave(final DataSetDef dataSetDef, final String message) {
 
     RemoteCallback<Path> saveSuccessCallback = new RemoteCallback<Path>() {
         @Override public void callback(Path path) {
+            currentWorkflow.clear();
```

Comment:
@dgutierr I dug some more into this difference and I think this is it: kie-wb's entry point has [UncaughtExceptionHandler|https://github.com/kiegroup/kie-wb-common/blob/master/kie-wb-common-screens/kie-wb-common-workbench/kie-wb-common-workbench-client/src/main/java/org/kie/workbench/common/workbench/client/entrypoint/DefaultWorkbenchEntryPoint.java#L80] which shows the error in the modal.